app/views/admins/show_facilities.html.erb

```diff
@@ -0,0 +1,12 @@
+Facilities
+
+<div>
+  <% @facilities.each do |facility| %>
+    <div>
+      <%=facility.name%> <%= facility.city %> <%= facility.state %> <%= facility.zip %> 
+          <%= facility.specialty %> 
+      <%= link_to "Edit", edit_facility_path(facility.id), method: :'get' %>
```

Comment:
You don't need to specify `method: :get`  in any of your links. That's the default for the given path.

Reply:
Removed method: :'get'.